client/signup/main.jsx

```diff
@@ -41,6 +41,8 @@ import utils from './utils';
 import { currentUserHasFlag, getCurrentUser } from 'state/current-user/selectors';
 import { DOMAINS_WITH_PLANS_ONLY } from 'state/current-user/constants';
 import * as oauthToken from 'lib/oauth-token';
+import DocumentHead from 'components/data/document-head';
+import i18n from 'i18n-calypso';
```

Comment:
There is no need to import the whole `i18n-calypso` library here, when we're just using `translate`.

It can be done like this:

```
import { translate } from 'i18n-calypso';
...
return this.props.flowName === accountFlowName ? translate( 'Create an account' ) : translate( 'Create a site' );
```